src/pages/storage/StoragePoolDetail.tsx

```diff
@@ -51,10 +54,10 @@ const StoragePoolDetail: FC = () => {
     return <>Loading storage details failed</>;
   }
 
-  const tabs: (string | TabLink)[] = [
-    "Overview",
-    "Configuration",
-    {
+  const supportedStorageDrivers = getSupportedStorageDrivers(settings);
+  const tabs: (string | TabLink)[] = ["Overview", "Configuration"];
+  if (supportedStorageDrivers.has(pool.driver)) {
+    tabs.push({
```

Comment:
I think we can have the volume link also for unsupported storage drivers. No need to limit it here.